### PR TITLE
{numlib}[GCC/11.2.0] BLIS v3.0.1 (AMD fork)

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc060 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/8239e9f38519bbac2d1769a7804cf507 for a full test report.

File: easybuild/easyconfigs/b/BLIS/BLIS-3.0.1-GCCcore-11.2.0-amd.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+homepage = 'https://developer.amd.com/amd-cpu-libraries/blas-library/'
+description = """AMD's fork of BLIS. BLIS is a portable software framework for instantiating high-performance
+BLAS-like dense linear algebra libraries."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/amd/blis/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['dff643e6ef946846e91e8f81b75ff8fe21f1f2d227599aecd654d184d9beff3e']
```
Comment (akesandgren) [inline easybuild/easyconfigs/b/BLIS/BLIS-3.0.1-GCCcore-11.2.0-amd.eb:15]: Are you sure you don't need the broadwell dgemm patch here too?

Comment (SebastianAchilles) [reply easybuild/easyconfigs/b/BLIS/BLIS-3.0.1-GCCcore-11.2.0-amd.eb]: No, I am not sure. But personally I would only use the AMD fork on AMD systems.
Do you think it makes sense to add the broadwell here as well?

Comment (akesandgren) [reply easybuild/easyconfigs/b/BLIS/BLIS-3.0.1-GCCcore-11.2.0-amd.eb]: The BLIS PR mentioned something about broadwell and zen2 having some common code. And the broadwell dgemm patch removed code that shouldn't be there so I guess it's rather safe...

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/999c001cf85e05fdfa2b6689ca83ff6e for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/b72aa465b0b9fd10940c35d21c8bfdb7 for a full test report.

File: easybuild/easyconfigs/b/BLIS/BLIS-3.0.1-GCCcore-11.2.0-amd.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/b/BLIS/BLIS-3.0.1-GCCcore-11.2.0-amd.eb]: broadwell patch added

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.3
See https://gist.github.com/7e475af54822725f0a1b1cb5cf916c40 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT="16 --gres=gpu:1"

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node3108.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/644c2484bb457636a0c3affd45ba81c0 for a full test report.

**Comment by boegel** [comment]
@SebastianAchilles Not sure if we should care that this fails on an Intel Skylake system?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3502.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/62802f44c9aeb9a37f8bfc49e5de7ffe for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**